docs/modelsearch.rst

```diff
@@ -392,6 +392,8 @@ MFL support the following types of options to feature descriptions:
 +---------------+------------------+-------------------------------------------------------+
 | array         | :code:`[FO, ZO]` | Multiple tokens or numbers                            |
 +---------------+------------------+-------------------------------------------------------+
+| symbol        | :code:`@IIV`     | A symbol with manual or automatic definition          |
++---------------+------------------+-------------------------------------------------------+
 
 Model features
```

Comment:
I was thinking since the MFL now covers more than one tool (and the idea is to extend to IIV etc, right?), then we can have this documentation in Tools under the subheading Common features?

Reply:
Yep that makes sense. I will move it.